src/app/games/lodestone/lodestone.component.ts

```diff
@@ -188,7 +188,18 @@ export class LodestoneComponent
         const move: LodestoneMove = new LodestoneMove(coord, lodestone.direction, lodestone.orientation, this.captures);
         return this.chooseMove(move, this.getState());
     }
-    public async selectPressurePlate(position: LodestonePressurePlatePosition, index: number): Promise<MGPValidation> {
+    public selectOrDeselectPressurePlate(
```

Comment:
is it our convention :p ?

Reply:
j'sais plus :D Mais j'ai monoligné après le rename (sauf le type sur la ligne suivante)

Reply:
Faudra créer ce "nos conventions.md" ou je ne sais quoi :p tu veux le ticketter :p ?
Temporairement le ticket "sera le fichier" :p

Reply:
créed! Faudra y remplir petit à petit